spynnaker8/__init__.py

```diff
@@ -128,6 +132,7 @@
     # plastic stuff
     'STDPMechanism', 'AdditiveWeightDependence',
     'MultiplicativeWeightDependence', 'SpikePairRule',
+    'SpikeNearestPairRule',
```

Comment:
I think this already exists in spynnaker8.extra_models.__init__.py, though I think it has a different name; it should be renamed there if necessary, but shouldn't be in the default rule set as it is an extra one...